aries-site/src/pages/index.js

```diff
@@ -32,10 +32,11 @@ const Index = () => (
             Mix, match, and stack cards to change the game
           </Heading>
           <Paragraph size="large" fill>
-            The Carte Design System was created to empower designers, developer,
-            and others to contribute in making great experiences for the
-            customer. Carte means “card game” in Italian. Pick your cards and
-            see what games you can play.
+            The Design System was created to empower designers,
+            developer, and others to contribute in making
+            great experiences for the customer.{' '}
+            {/* Carte means “card game” in Italian. Pick your cards and 
```

Comment:
Is there benefit to leaving this commented out? To keep the code clean, should we just remove this for now?

Reply:
I was just leaving it commented out in case we wanted to use the same sentence but we can take it out either way is fine with me

Reply:
I'd say let's take it out for now since we can always go back in the commits/PRs to get that sentence if we want it

Reply:
sounds good!